lib/solrb/request/limit_docs_by_field.rb

```diff
@@ -2,16 +2,20 @@ module Solr
   class Request
```

Comment:
this is a custom solr plugin that shouldn't be here, it should remain in Machinio

Reply:
got it